src/Microsoft.ML.TimeSeries/IidChangePointDetector.cs

```diff
@@ -192,16 +192,9 @@ private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, Dat
     }
 
     /// <summary>
-    /// Estimator for <see cref="IidChangePointDetector"/>
+    /// The <see cref="IEstimator{ITransformer}"/> for detecting a signal change on an Independent Identically Distributed time series.
```

Comment:
>IEstimator{ITransformer} [](start = 23, length = 24)

Why this changed?

Reply:
We're standardizing the documentation across types, and i added a bit more info too. 

---
In reply to: [264467232](https://github.com/dotnet/machinelearning/pull/2900#discussion_r264467232) [](ancestors = 264467232)